app/src/main/java/lt/getpet/getpet/PetProfileActivity.kt

```diff
@@ -78,16 +81,30 @@ class PetProfileActivity : BaseActivity() {
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         if (item.itemId == R.id.delete_pet) {
-            val disposable = petsService.savePetChoice(pet, false)
-                    .subscribeOn(ioScheduler)
-                    .observeOn(uiScheduler)
-                    .subscribe({
-                        finish()
-                    }, {
-                        Timber.w(it)
-                    })
-
-            subscriptions.add(disposable)
+            val builder = AlertDialog.Builder(this, R.style.MyDialogTheme)
+            builder.setTitle(R.string.dialog_delete_title)
+                    .setMessage(R.string.dialog_delete_message)
+                    .setPositiveButton(R.string.button_ok
+                    ) { dialog, id ->
+                        val disposable = petsService.savePetChoice(pet, false)
+                                .subscribeOn(ioScheduler)
+                                .observeOn(uiScheduler)
+                                .subscribe({
+                                    finish()
+                                }, {
+                                    Timber.w(it)
+                                })
+                        subscriptions.add(disposable)
+                    }
+                    .setNegativeButton(R.string.button_cancel
+                    ) { dialog, id ->
```

Comment:
Most of the time in Kotlin when variable is never used we rename it to `_`. IDE should give hint for this.

Reply:
Yes, IDE certainly  it gives hints for that. Fixed.